common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -20,6 +20,8 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
```

Comment:
Why add line 23, 24 ?

Reply:
This was added by an earlier change that was removed later. They are redundant now so I removed them in the newest commit.